mlrun/run.py

```diff
@@ -1526,6 +1526,12 @@ def log_plot(ctx: MLClientCtx, obj, key: str, logging_kwargs: dict):
                     artifact = BokehArtifact(key=key, figure=obj)
             except ModuleNotFoundError:
                 pass
+            except ImportError:
+                logger.warn(
+                    "Bokeh installation is ignored. If needed, "
+                    "make sure you have the required version with `pip install mlrun[bokeh]`"
+                )
+                pass
```

Comment:
No need to pass when there is a line of code in this code segment (use `pass` when no code is executed yet you need to fill the indention so python's parser will know how to read your code).
```suggestion
               
```

Reply:
oops nice catch